### PR TITLE
Coveralls ci

### PR DESCRIPTION
**Pull request by P33J3**
coveralls

screenshot showing a successful run of coverall

screencapture-app-circleci-pipelines

screenshot showing a successful run of circle CI 

Notes:

I am not sure if the app was truly deployed to Heroku.

I am not sure how to update the coverage badge on Github.

Circle CI is now pushing to Coverall but coverage is not updating

### CODE REVIEW

**Comment by P33J3** [comment]
In last commit, i used https://circleci.com/developer/orbs/orb/coveralls/coveralls to try to push coverage from CircleCI to Coveralls. Code has been added to .circleci/config.yml

File: coverage/clover.xml [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        <line num="24" count="1" type="stmt"/>
+        <line num="27" count="1" type="stmt"/>
+        <line num="30" count="1" type="stmt"/>
+        <line num="31" count="0" type="stmt"/>
+        <line num="32" count="0" type="stmt"/>
+        <line num="33" count="0" type="stmt"/>
+        <line num="34" count="0" type="stmt"/>
+        <line num="35" count="0" type="stmt"/>
+        <line num="37" count="0" type="stmt"/>
+        <line num="40" count="1" type="stmt"/>
```
Comment (Avuidrauxs) [inline coverage/clover.xml:23]: ### Critical

Add the coverage directory to `.gitignore`

File: logs/all.log [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+2022-04-30 21:40:35:4035 [37mdebug[39m: [37mThis is a debug log[39m
+2022-04-30 21:40:35:4035 [35mhttp[39m: [35mGET /logger 304 - - 5.787 ms[39m
+2022-04-30 23:23:34:2334 [37mdebug[39m: [37mServer is up and running @ http://localhost:3000[39m
+2022-05-01 01:09:25:925 [37mdebug[39m: [37mServer is up and running @ http://localhost:3000[39m
+2022-05-01 01:09:34:934 [31merror[39m: [31mThis is an error log[39m
+2022-05-01 01:09:34:934 [33mwarn[39m: [33mThis is a warn log[39m
+2022-05-01 01:09:34:934 [32minfo[39m: [32mThis is a info log[39m
+2022-05-01 01:09:34:934 [35mhttp[39m: [35mThis is a http log[39m
+2022-05-01 01:09:34:934 [37mdebug[39m: [37mThis is a debug log[39m
+2022-05-01 01:09:34:934 [35mhttp[39m: [35mGET /logger 304 - - 6.038 ms[39m
```
Comment (Avuidrauxs) [inline logs/all.log]: ### Critical

Add the logs directory to `.gitignore`

**Review by Avuidrauxs — CHANGES_REQUESTED** [review]
Awesome job putting together Coveralls and CircleCI, you seem to have a thing for DevOps 😅 . Resolve the critical comments before we proceed on this PR.

**Pull request merged**